dataUtility.py

```diff
@@ -10,7 +10,7 @@
 
 class csV2Cla():
 
-    def __init__(self, fname = None, fout = None, fpickle = None, fcount = None):
+    def __init__(self, fname = None, fpickle = None):
```

Comment:
Maybe start an if statement at the beginning to check for either fname or fpickle. If fname is provided then do all of this processing and save self at the end. If fpickle is provided, just make self = loaded self and skip all of that

Reply:
Sorry I need to meet with another group, I suggest merging it and then fixing it on your next commit

Reply:
wait, I do have if statements for that, there's one to make an empty object for the loading function, however I have fname and fpickle in there, for creation and storing, do you mean calling the load function from init? because that might be a bit harder.

Reply:
Oh, I see now, it just returns if fname is None and then you call load afterwards. That works!